src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-panel" data-qa="big">
+      <label for="bigSearch"></label>
+      <img
+      class="icon"
+      src="images/Search.svg"
+      alt="icon1">
+      <input
+      type="text"
+      id="bigSearch" class="search-bar"
+      data-qa="keypress"
+      placeholder="Try “Los Angeles”">
```

Comment:
```suggestion
      <img
        class="icon"
        src="images/Search.svg"
        alt="icon1"
      >
      <input
        type="text"
        id="bigSearch" class="search-bar"
        data-qa="keypress"
        placeholder="Try “Los Angeles”"
      >
```
Consider fixing indents according to the code style guide